snap/local/bin/source-env-file.sh

```diff
@@ -1,13 +1,13 @@
-#!/bin/bash -e
+#!/bin/bash -ex
```

Comment:
```suggestion
#!/bin/bash -e
```

I think if xtrace is enabled, the commands will appear in syslogs without the appropriate component name or tag. The reader can't know which application they come from.